client/src/common/components/Header.tsx

```diff
@@ -1,87 +1,89 @@
+            Log in
+          </button>
+        </Link>
+        <Link to="/signup">
+          <button className="font-bold text-header_color border-2 border-solid	border-primary_bg_color bg-primary_bg_color rounded-full px-3 text-sm">
+            Sign up
+          </button>
+        </Link>
+      </div>
+    );
+  };
 
-	const closeClickingOutsideMenu = (e: any) => {
-		if (e.target.closest('.menu') === null) {
-			setOpen(false);
-		}
-	};
-
-	useEffect(() => {
-		document.addEventListener('click', closeClickingOutsideMenu);
-	}, []);
-
-	return (
-		<>
-			<header className="bg-header_color w-screen h-mobileHeaderHeight md:h-laptopHeaderHeight fixed z-10 menu">
-				<div className="flex items-center	font-sans text-primary_bg_color h-full mx-4">
-					<div className="flex-grow">
-						<Link to="/" className="italic font-bold">
-							Senpai-Kohai
-						</Link>
-					</div>
-					{headerItems()}
-				</div>
-			</header>
-			<div
-				className="w-screen bg-tertiary_bg_color absolute top-mobileHeaderHeight flex justify-center items-center transition duration-500 ease-linear transform menu"
-				style={menuStyle()}
-			>
-				<ul className="text-header_color font-sans font-bold flex flex-col gap-y-4 text-xl">
-					<Link to="/board" onClick={() => setOpen(false)}>
-						<li>Browsing Peeps</li>
-					</Link>
-					<Link to="/profile/senpai" onClick={() => setOpen(false)}>
-						<li>My Profile as Senpai</li>
-					</Link>
-					<Link to="/profile/kohai" onClick={() => setOpen(false)}>
-						<li>My Profile as Kohai</li>
-					</Link>
-				</ul>
-			</div>
-		</>
-	);
+  return (
+    <>
+      <header className="bg-header_color w-screen h-mobileHeaderHeight md:h-laptopHeaderHeight fixed z-30 menu">
+        <div className="flex items-center	font-sans text-primary_bg_color h-full mx-4">
+          <div className="flex-grow">
+            <Link to="/" className="italic font-bold">
+              Senpai-Kohai
+            </Link>
+          </div>
+          {headerItems()}
+        </div>
+      </header>
+      <div
+        className="w-screen bg-tertiary_bg_color fixed top-mobileHeaderHeight flex justify-center items-center transition duration-500 ease-linear transform menu z-20"
+        style={menuStyle()}
+      >
+        <ul className="text-header_color font-sans font-bold flex flex-col gap-y-4 text-xl">
+          <Link to="/board" onClick={() => setOpen(false)}>
+            <li>Browsing Peeps</li>
+          </Link>
+          <Link to="/profile/senpai" onClick={() => setOpen(false)}>
+            <li>My Profile as Senpai</li>
+          </Link>
+          <Link to="/profile/kohai" onClick={() => setOpen(false)}>
+            <li>My Profile as Kohai</li>
+          </Link>
+        </ul>
+      </div>
+      <div
+        className={[
+          "z-10",
+          "w-full",
+          "h-screen",
+          // "bg-gray-300",
+          // "opacity-50",
```

Comment:
```suggestion
          'backdrop-filter',
	  'backdrop-blur-sm',
```
**Suggestion**: make the background blurry instead.
![スクリーンショット 2021-11-29 11 10 23](https://user-images.githubusercontent.com/51708229/143928302-fc50762a-133b-4000-8aaa-7a5bbd10a682.png)